working-with-openai/setting-up-openai-postman.md

```diff
@@ -0,0 +1,91 @@
+## Setting up the OpenAI API in Postman
+
+## Goals
+Now that we have our OpenAI account set up and verified, let's  get OpenAI set up in Postman and Python so we are prepared to start testing our API calls and bringing them into our project.
+
+Our goals for this lesson are to:
+- Set up an OpenAI environment in Postman
+- Make our first Postman API 
+- Learn how to set up the OpenAI library in Python
```

Comment:
Is this content meant to go in another page? I'm wondering if this bullet should be removed since I didn't see materials for using OpenAI with Python/in VS Code in this lesson.